core/src/main/java/com/linecorp/armeria/client/proxy/HAProxyConfig.java

```diff
@@ -67,6 +67,12 @@ public InetSocketAddress sourceAddress() {
         return sourceAddress;
     }
 
+    @Override
+    public ProxyConfig withNewProxyAddress(InetSocketAddress newProxyAddress) {
+        return this.sourceAddress == null ? new HAProxyConfig(proxyAddress)
+                                          : new HAProxyConfig(proxyAddress, this.sourceAddress);
```

Comment:
The input is ignored.
```suggestion
        requireNonNull(newProxyAddress, "newProxyAddress");
        return this.sourceAddress == null ? new HAProxyConfig(newProxyAddress)
                                          : new HAProxyConfig(newProxyAddress, this.sourceAddress);
```

Reply:
This comment wasn't addressed. Was it intended? Let me know if I'm missing something or misunderstood.

Reply:
I'm really sorry about this.  🙇‍♂️ 
I missed it. 
Now, I fixed it!